lib/elixir/lib/string.ex

```diff
@@ -1812,4 +1812,91 @@ defmodule String do
       {comm + 1, trans, current}
     end
   end
+
+  @spec difference(t, t) :: [{:eq | :ins | :del, t}] | nil
+  def difference(str1, str2) do
```

Comment:
I didn't read the paper yet :), but yeah it seems fair enough. I just feel that the "diff" is a concept as a whole, even if it's just an abbreviation of difference.